app/src/test/java/org/simple/clinic/summary/PatientSummaryScreenControllerTest.kt

```diff
@@ -565,8 +566,18 @@ class PatientSummaryScreenControllerTest {
     verify(medicalHistoryRepository).save(eq(updatedMedicalHistory), any())
   }
 
-  private fun randomAnswer(): MedicalHistory.Answer {
-    return MedicalHistory.Answer::class.java.enumConstants.asList().shuffled().first()
+  @Suppress("unused")
+  fun medicalHistoryQuestionsAndAnswers(): List<List<Any>> {
+
+    fun randomAnswer(): MedicalHistory.Answer {
+      return MedicalHistory.Answer::class.java.enumConstants.asList().shuffled().first()
```

Comment:
Oh wow. Did you intentionally not use `MedicalHistory.Answer.values()` in favor of `MedicalHistory.Answer::class.java.enumConstants`?

Reply:
I didn't write this function, moved it to a place where it was being used. I'll fix this.